doc/radosgw/config-ref.rst

```diff
@@ -175,23 +175,23 @@ instances or all radosgw-admin commands can be put into the ``[global]`` or the
 :Default: ``30``
 
 
-``rgw mime types file``
+``rgw_mime_types_file``
 
-:Description: The path and location of the MIME types. Used for Swift
+:Description: The path and location of the MIME types file. Used for Swift
```

Comment:
```suggestion
:Description: The path and location of the MIME-types file. Used for Swift
```

Reply:
I'm less convinced re this one, but not enough to dissent